.eslintrc

```diff
@@ -2,26 +2,21 @@
   "env": {
     "node": true,
     "mocha": true,
-    "es6": true
+    "es6": true,
   },
   "parser": "@typescript-eslint/parser",
   "parserOptions": {
     "ecmaFeatures": {
-      "impliedStrict": true
-    }
+      "impliedStrict": true,
+    },
   },
   "plugins": ["@typescript-eslint", "prettier"],
   "rules": {
     "security/detect-non-literal-regexp": 0,
     "no-unused-expressions": 0,
     "@typescript-eslint/no-explicit-any": 0,
     "@typescript-eslint/ban-types": 0,
-    "comma-dangle": 0
+    "comma-dangle": 0,
   },
-  "extends": [
-    "eslint:recommended",
-    "plugin:@typescript-eslint/recommended",
-    "plugin:prettier/recommended",
-    "emarsys"
-  ]
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended", "emarsys", "plugin:prettier/recommended"],
```

Comment:
put prettier config at the end to disable the formatting rules of eslint enabled by the emarsys config.